lecompteurdelaroulotte/app/src/main/res/layout/fragment_rule.xml

```diff
@@ -194,7 +196,7 @@
         android:layout_height="match_parent"
         android:layout_alignParentStart="true"
         tools:listitem="@layout/list_step_item"
-        android:layout_marginTop="236dp"
+        android:layout_marginTop="246dp"
```

Comment:
This line always bother me because we were forced to choose a value that would fit the elements above the ListView because the layoutConstraints doesn't allow much flexibility. I know there aren't much solution beside refractor a whole bunch of things but we should look for a solution in the future (maybe future PR ?)